### PR TITLE
Add Kerio-Connect.app v9.2.5.9921

### PR DESCRIPTION
**Pull request by ahousseini**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/kerio-connect.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'kerio-connect' do
+  version '9.2.5.9921'
+  sha256 '01b1c9da268cdccbbba560e22e2d615e37e4ca4818ac57dc84c598949b6f9a6a'
+
+  # kerio-dc-releases.kerio.com was verified as official when first introduced to the cask
+  url "http://kerio-dc-releases.kerio.com/kerio-connect-client-update-darwin64-#{version}.zip"
```
Comment (commitay) [inline Casks/kerio-connect.rb]: Where did you get this URL, the support page has a different one?

Comment (ahousseini) [reply Casks/kerio-connect.rb]: If the 9.2.5.9920 version is installed, the application always reports an update without performing the update. I contacted the support and they gave me this URL. I don't know why version 9.2.5.9921 is not listed on the support page.

Comment (commitay) [reply Casks/kerio-connect.rb]: The support page has a different version and URL:

http://cdn.kerio.com/dwn/connect/connect-9.2.5-3336/kerio-connect-9.2.5-3336-p3-mac.dmg

Comment (ahousseini) [reply Casks/kerio-connect.rb]: Oh, that's the server. My pull request is for the client. Should we rename the cask?

Comment (commitay) [reply Casks/kerio-connect.rb]: Apologies, I didn't see the tabs for the different types on the support page.

> Should we rename the cask?

No, `kerio-connect` is fine. If the server is added we can call it `kerio-connect-server`

**Review by commitay — APPROVED** [review]

**Pull request merged**